### PR TITLE
chore(starters): add gatsby-portfolio-starter

### PR DESCRIPTION
**Pull request by Judionit**
a simple portfolio starter

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -5199,3 +5199,14 @@
     - Disqus
     - Resume
     - Place plan on the top
+- url: https://gatsby-portfolio-starter.netlify.com/
+  repo: https://github.com/Judionit/gatsby-portfolio-starter
+  description: A simple Gatsby portfolio starter
+  tags:
+    - Netlify
+    - Styling: CSS-in-JS
```
Comment (amandeepmittal) [inline docs/starters.yml]: Can you please change this tag from `Styling: CSS-in-JS` to `Styling:CSS-in-JS` such that it reflects the correct tag?

**Comment by Judionit** [comment]
changed to Style:CSS-in-JS

**Comment by Judionit** [comment]
why my starter is not merge yet ? is something wrong ?

**Comment by amandeepmittal** [comment]
Hey @Judionit someone with authorization to merge this PR will be get back to you.

**Comment by laurieontech** [comment]
Thanks for the PR! README looks great :)

**Comment by ahmadawais** [comment]
@laurieontech offtopic, can you share with me how you folks name a starter? The name param is not the part of the yml file. Is there a file that has all the names of starters with their info?

**Comment by gillkyle** [comment]
> offtopic, can you share with me how you folks name a starter? The name param is not the part of the yml file. Is there a file that has all the names of starters with their info?

Hey @ahmadawais, if you're referring to how starters are named in places like the [starter library](https://www.gatsbyjs.org/starters/?v=2) that info is taken from the repository url. Ie the name for gatsby-starter-blog is just the latter end of the url https://github.com/gatsbyjs/gatsby-starter-blog.

**Comment by ahmadawais** [comment]
Silly me, thanks for sharing that, I was hoping to do that myself but didn't know it's the slug that becomes the name, it's already done. Appreciate the help on this @gillkyle 🙌

**Pull request merged**